tarorpc/taro.proto

```diff
@@ -285,6 +290,53 @@ message ListBalancesResponse {
     map<string, AssetFamilyBalance> asset_family_balances = 2;
 }
 
+message ListTransfersRequest {
+}
+
+message ListTransfersResponse {
+    // The unordered list of outgoing asset transfers.
+    repeated AssetTransfer transfers = 1;
+}
+
+message AssetTransfer {
+    int64 transfer_timestamp = 1;
+
+    // The old/current location of the Taro commitment that was spent as an
+    // input.
+    string old_anchor_point = 2;
+
+    // The new location of the Taro commitment referenced by the old anchor
+    // point.
+    string new_anchor_point = 3;
+
+    // The new Taro root that commits to the set of modified and unmodified
+    // assets.
+    bytes taro_root = 4;
+
+    // The new transaction that commits to the set of Taro assets found at the
+    // above new anchor point.
+    bytes anchor_tx_hash = 5;
+
+    // Describes the set of mutated assets that now live at the new anchor tx
+    // point.
+    repeated AssetSpendDelta asset_spend_deltas = 6;
+}
+
+message AssetSpendDelta {
```

Comment:
In the future I think we also want to add more information here like splits, etc -- but this is a solid base to build off of.